From: TheLazieR Yip <thelazier@gmail.com>
Date: Wed, 9 Mar 2016 11:11:11 +0700
Subject: [PATCH] Bump openssl to 1.0.1t

---
 depends/packages/openssl.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/depends/packages/openssl.mk b/depends/packages/openssl.mk
--- a/depends/packages/openssl.mk
+++ b/depends/packages/openssl.mk
@@ -1,8 +1,8 @@
 package=openssl
-$(package)_version=1.0.1k
+$(package)_version=1.0.1t
 $(package)_download_path=https://www.openssl.org/source
 $(package)_file_name=$(package)-$($(package)_version).tar.gz
-$(package)_sha256_hash=8f9faeaebad088e772f4ef5e38252d472be4d878c6b3a2718c10a4fcebe7a41c
+$(package)_sha256_hash=4a6ee491a2fdb22e519c76fdc2a628bb3cec12762cd456861d207996c8a07088
 
 define $(package)_set_vars
 $(package)_config_env=AR="$($(package)_ar)" RANLIB="$($(package)_ranlib)" CC="$($(package)_cc)"
